src/models/units.ts

```diff
@@ -154,6 +197,31 @@ export const units: Units = {
         phenotypeHeading: "Pea type",
         getPhenotypeLabel: (phenotype) => phenotype.charAt(0).toUpperCase() + phenotype.slice(1),
         getGenotypeHTMLLabel: (genotype) => genotype,
+        chartTypes: {
+          phenotype: {legend: [
+            {label: "Smooth", color: colors.colorDataMouseBrownLightRep},
+            {label: "Wrinkled", color: colors.colorDataMouseBrownMediumRep}
+          ], title: "Pea Types" },
+          genotype: {legend: [
+            {label: "RR Peas", color: colors.colorDataMouseBrownLightRep},
+            {label: "Rr Peas", color: colors.colorDataMouseBrownMediumRep},
+            {label: "rR Peas", color: colors.colorDataMouseBrownMediumRep},
+            {label: "rr Peas", color: colors.colorDataMouseBrownDarkRep}
```

Comment:
I assume these will eventually be more "pea-specific" colors, but we don't have spec yet?